### PR TITLE
[Profile] Edit paymento info

### PR DESCRIPTION
**Pull request by lemefe21**

### CODE REVIEW

File: sdk/src/main/java/com/ingresse/sdk/services/UserService.kt [hunk 1]
```diff
@@ -205,6 +211,70 @@ class UserService(private val client: IngresseClient) {
         mUpdateBasicInfosCall?.enqueue(RetrofitCallback(type, callback))
     }
 
+    /**
+     * Update user basic infos
```
Comment (PhillipiLino) [inline sdk/src/main/java/com/ingresse/sdk/services/UserService.kt]: Update user address

File: sdk/src/main/java/com/ingresse/sdk/services/UserService.kt [hunk 2]
```diff
@@ -205,6 +211,70 @@ class UserService(private val client: IngresseClient) {
...
+                          onError: ErrorBlock,
+                          onConnectionError: (Throwable) -> Unit,
+                          onTokenExpired: Block) {
+
+        mUpdateUserAddressCall = service.updateUserAddress(
+                userId = request.userId,
+                apikey = client.key,
+                userToken = request.userToken,
+                params = request,
+                method = "update"
```
Comment (PhillipiLino) [inline sdk/src/main/java/com/ingresse/sdk/services/UserService.kt]: We don't use `method` anymore

File: sdk/src/main/java/com/ingresse/sdk/services/UserService.kt [hunk 3]
```diff
@@ -205,6 +211,70 @@ class UserService(private val client: IngresseClient) {
...
+                    val apiError = APIError()
+                    apiError.message = response.message.joinToString(", ")
+                    apiError.title = "Verifique suas informações"
+                    apiError.code = 0
+                    onError(apiError)
+                    return
+                }
+
+                if (response.status == null) return onError(APIError.default)
+
```
Comment (PhillipiLino) [inline sdk/src/main/java/com/ingresse/sdk/services/UserService.kt]: Do something like this
```kotlin
if (response.status == null) return onError(APIError.default)
                val responseData = response.data ?: return onError(APIError.default)
                if (!response.status) return onError(APIError.default)
                onSuccess(responseData)
```

File: sdk/src/main/java/com/ingresse/sdk/request/User.kt [hunk 4]
```diff
@@ -32,6 +33,20 @@ interface User {
                          @Query("usertoken") userToken: String,
                          @Body params: UserBasicInfos): Call<String>
 
+    /**
+     * Update user address
+     *
+     * @param userId - id from user
+     * @param method - info request action
```
Comment (tmikulsk1) [inline sdk/src/main/java/com/ingresse/sdk/request/User.kt]: ?

File: sdk/src/main/java/com/ingresse/sdk/services/UserService.kt [hunk 5]
```diff
@@ -205,6 +211,66 @@ class UserService(private val client: IngresseClient) {
...
 
+    /**
+     * Update user address
+     *
+     * @param request - all parameters used for update address
+     * @param onSuccess - success callback
+     * @param onError - error callback
+     * @param onConnectionError - connection error callback
+     * @param onTokenExpired - token expired error callback
+     * @param onConnectionError - connection error callback
```
Comment (tmikulsk1) [inline sdk/src/main/java/com/ingresse/sdk/services/UserService.kt]: Duplicated here.

File: sdk/src/main/java/com/ingresse/sdk/services/UserService.kt [hunk 6]
```diff
@@ -205,6 +211,66 @@ class UserService(private val client: IngresseClient) {
...
+                }
+
+                if (response.status == null) return onError(APIError.default)
+
+                val responseData = response.data ?: return onError(APIError.default)
+                if (!response.status) return onError(APIError.default)
+
+                onSuccess(responseData)
+            }
+            override fun onError(error: APIError) = onError(error)
```
Comment (tmikulsk1) [inline sdk/src/main/java/com/ingresse/sdk/services/UserService.kt]: Put one line here.

File: sdk/src/main/java/com/ingresse/sdk/services/ZipCodeService.kt [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+        service = adapter.create(ZipCode::class.java)
+    }
+
+    /**
+     * Method to cancel search address
+     */
+    fun cancelRecentTransfersData() = mZipCodeCall?.cancel()
+
+    /**
+     * Method to search address by zipcode
```
Comment (tmikulsk1) [inline sdk/src/main/java/com/ingresse/sdk/services/ZipCodeService.kt]: You forgot to write **@param**'s.

**Pull request closed without merging**